librad/src/git/p2p/url.rs

```diff
@@ -165,27 +165,23 @@ impl Into<RadUrl> for GitUrl {
 
 #[derive(Debug, PartialEq)]
 pub struct GitUrlRef<'a> {
-    pub local_peer: &'a PeerId,
-    pub remote_peer: &'a PeerId,
+    pub local_peer: PeerId,
```

Comment:
Does clippiyeahyeah complain about using references here? I'd expect these `*Ref<'a>` types to be "zero-cost" and not do any cloning or copying.

Reply:
Nope, I was overzealous

Reply:
Reverted those changes